pkg/processor/trigger/kafka/trigger.go

```diff
@@ -364,11 +279,115 @@ func (k *kafka) ConsumeClaim(session sarama.ConsumerGroupSession, claim sarama.C
 	close(submittedEventChan)
 	close(explicitAckControlMessageChan)
 	close(workerDrainingCompleteChan)
-	close(readyForRebalanceChan)
 
 	return submitError
 }
 
+func (k *kafka) drainOnRebalance(session sarama.ConsumerGroupSession,
+	claim sarama.ConsumerGroupClaim,
+	workerInstance *worker.Worker,
+	submittedEventInstance *submittedEvent,
+	message *sarama.ConsumerMessage,
+	finishHandlingMsg bool) {
+
+	k.Logger.DebugWith("Got signal to stop consumption",
```

Comment:
this log should be outside the function, right when we get `session.Context().Done()`, because it doesn't log what the function is doing, more why it was called.
If you want to log here it should say "Draining on rebalance".